proto/cosmos/group/v1beta1/types.proto

```diff
@@ -285,4 +291,7 @@ message Vote {
 
   // submitted_at is the timestamp when the vote was submitted.
   google.protobuf.Timestamp submitted_at = 5 [(gogoproto.nullable) = false, (gogoproto.stdtime) = true];
+
+  // choices is the split of votes.
+  repeated WeightedVoteOption choices = 6;
```

Comment:
We should not have both `choice` and `choices`. Let's remove this, and replace field 3 with choices.